utils/ort/src/main/kotlin/storage/HttpFileStorage.kt

```diff
@@ -67,6 +67,12 @@ class HttpFileStorage(
      */
     private val cacheMaxAgeInSeconds: Int = 0
 ) : FileStorage {
+    private val requestBuilder by lazy {
+        Request.Builder()
+            .headers(headers.toHeaders())
+            .cacheControl(CacheControl.Builder().maxAge(cacheMaxAgeInSeconds, TimeUnit.SECONDS).build())
+    }
```

Comment:
Why is there no risk of re-using a single instance of the builder for building multiple requests?

Reply:
Why should there be any risk? The actual request is only created upon the `build()` call, which still is performed in the individual functions.

Reply:
Because if on user of the builder alters the state, the altered state may apply to the other user?

Reply:
Ah, right. I was falsely assuming that each method call on the builder would return a new instance instead of modifying the existing one. Thanks, fixed.